src/main/java/de/tum/in/www1/artemis/domain/Exercise.java

```diff
@@ -100,6 +100,10 @@ public abstract class Exercise extends BaseExercise implements LearningObject {
     @Column(name = "second_correction_enabled")
     private Boolean secondCorrectionEnabled = false;
 
+    @Nullable
+    @Column(name = "feedback_suggestions_enabled") // enables Athena
+    private Boolean feedbackSuggestionsEnabled = false;
```

Comment:
```suggestion
    @Column(name = "feedback_suggestions_enabled") // enables Athena
    private boolean feedbackSuggestionsEnabled = false;
```
The database migration sets a default value, too. It does not need to be nullable.